handlers/register.go

```diff
@@ -0,0 +1,33 @@
+package handlers
+
+import (
+	"net/http"
+
+	"github.com/companieshouse/chs.go/authentication"
+	"github.com/companieshouse/chs.go/log"
+	"github.com/companieshouse/emergency-auth-code-api/config"
+	"github.com/gorilla/mux"
+)
+
+// Contains config and DAO for
```

Comment:
Comment should begin with name
```suggestion
// EmergencyAuthCodeService contains config and DAO for
```